src/Life.Tests/BoardTest.cs

```diff
@@ -6,25 +6,20 @@ namespace Life.Tests;
 
 public class BoardTest
 {
-    [Fact]
-    public void Should_Initialize_Board()
+    [Theory]
+    [MemberData(nameof(TestBoards.BoardScenarios), MemberType = typeof(TestBoards))]
```

Comment:
Reordered some things here to more logically follow the puzzle. Also parameterized the board tests so other test cases can be easily enabled